js/main.js

```diff
@@ -31,6 +31,31 @@ function getCSVData(callback = undefined) {
   });
 }
 
+/**
+ * Setup click events for hamburger nav
+ */
+function setupNavClickEvents () {
+  let mobileButton = document.getElementById("nav-button");
+  let closeButton = document.getElementById("close-nav-menu");
+  let navElement = document.getElementsByTagName("nav")[0];
+  function openNavMenu (event) {
+    this.classList.add("active");
+    setTimeout(() => {
+      this.classList.add("open");
+    }, 250);
```

Comment:
add a comment explaining the reason a timeout is needed, and if the 250 number is significant?